examples/test-controls/index.html

```diff
@@ -11,7 +11,7 @@
     <a-scene stats="true">
       <!-- We wrap the camera in an <a-entity> so we can position it without colliding with the HMD updates -->
       <a-entity>
-        <a-entity position="0 1.8 4" camera look-controls wasd-controls></a-entity>
+        <a-entity position="0 1.8 4" camera controls></a-entity>
```

Comment:
is `controls` the wrong word, do you think, or would you want to have more components explicitly declared in the markup?

Other ideas: `navigation`, `user-controls`, `camera-controls` .. those second two may be too specific, since this could presumably be moving something other than the camera.